src/sql/src/rbac.rs

```diff
@@ -767,6 +767,14 @@ fn generate_rbac_requirements(
             },
             ..Default::default()
         },
+        Plan::ExplainSchema(plan::ExplainSchemaPlan { sink_from, .. }) => RbacRequirements {
+            privileges: {
+                let item = catalog.get_item(sink_from);
+                let schema_id: ObjectId = item.name().qualifiers.clone().into();
+                vec![(SystemObjectId::Object(schema_id), AclMode::USAGE, role_id)]
```

Comment:
Was not sure what should be the required privileges here. Suggestions would be great!

Reply:
The `privileges` section looks good to me. I would also add `item_usage: &CREATE_ITEM_USAGE`.

Reply:
@jkosh44 should know for sure but I assume it should be the same as other `EXPLAIN` stuff.

Reply:
oops, didn't see he had already commented, disregard mine.

Reply:
updated